ui/pages/confirmations/hooks/useCurrentConfirmation.test.ts

```diff
@@ -70,6 +72,8 @@ function buildState({
       pendingApprovals: pendingApprovals ? arrayToIdMap(pendingApprovals) : {},
       preferences: {
         redesignedConfirmationsEnabled,
+        isRedesignedConfirmationsDeveloperEnabled:
+          isRedesignedConfirmationsDeveloperEnabled || false,
```

Comment:
I believe we can remove `|| false` here

Reply:
just defaulting it to false for all tests instead of leaving it undefined